assets/helpers/internationalisation/country.js

```diff
@@ -434,15 +432,25 @@ function handleCountryForCountryGroup(
 }
 
 function detect(countryGroupId: ?CountryGroupId = null): IsoCountry {
-  const country = handleCountryForCountryGroup('EURCountries', countryGroupId) ||
-                  handleCountryForCountryGroup('International', countryGroupId) ||
-                  handleCountryForCountryGroup('NZDCountries', countryGroupId) ||
-                  fromCountryGroup(countryGroupId) ||
-                  fromPath() ||
-                  fromQueryParameter() ||
-                  fromCookie() ||
-                  fromGeolocation() ||
-                  'GB';
+
+  const targetCountryGroups = ['International', 'EURCountries', 'NZDCountries', 'GBPCountries', 'AUDCountries'];
+  let country = null;
+
+  targetCountryGroups.forEach((targetCountryGroupId) => {
+    const candidateCountry = handleCountryForCountryGroup(targetCountryGroupId, countryGroupId);
+    if (candidateCountry !== null && country === null) {
+      country = candidateCountry;
+    }
+  });
+
+  if (country === null) {
+    country = fromCountryGroup(countryGroupId) ||
+      fromPath() ||
```

Comment:
This precedence seems right to me, however given my comment above, let's also discuss.